packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.CustomCallout.Example.tsx

```diff
@@ -27,7 +27,7 @@ export const HorizontalBarChartCustomCalloutExample: React.FunctionComponent<{}>
           horizontalBarChartdata: { x: 800, y: 15000 },
```

Comment:
All hover values are not fixed. one, seven, eight are still showing old value

Reply:
Fixed accordingly